src/natives/java_lang.ts

```diff
@@ -919,9 +919,14 @@ class java_lang_StrictMath {
   }
 
   public static 'log1p(D)D'(thread: threading.JVMThread, d_val: number): number {
-    thread.throwNewException('Ljava/lang/UnsatisfiedLinkError;', 'Native method not implemented.');
-    // Satisfy TypeScript return type.
-    return 0;
+    if(d_val < -1 || Number.isNaN(d_val))
```

Comment:
Use `isNaN`. Older browsers don't define `isNaN` on `Number`, and all browsers define it globally.